CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 6.14.0 - 2022-09-6
+### Changed
+- Support new rules introduced between rubocop 1.26 -> 1.3.6
```

Comment:
Should this say `1.30 -> 1.36` based on the lockfile changes?

Reply:
New cops are only enabled by default on new major releases. We havn't enabled any new cops since 1.26 based on our default.yml

Reply:
ok, then it should at least say `1.36` instead of `1.3.6`